ai-chat-assistant/autokitteh.yaml

```
@@ -0,0 +1,20 @@
+version: v1
+
+project:
+  name: ai_chatbot_assistant
+  vars:
+    - name: SHEET_ID
+      value:
+    - name: SHEET_NAME
+      value:
+    - name: REPO_NAME
+      value:
+  connections:
```

Cosmetic nit: see #203
```suggestion
  name: ai_chatbot_assistant

  vars:
    - name: SHEET_ID
      value:
    - name: SHEET_NAME
      value:
    - name: REPO_NAME
      value:

  connections:
```